pkg/microservice/aslan/core/common/service/helm.go

```diff
@@ -105,3 +123,26 @@ func preLoadServiceManifestsFromSource(svc *commonmodels.Service) error {
 
 	return nil
 }
+
+func preLoadServiceManifestsFromGerrit(svc *commonmodels.Service) error {
+	base := path.Join(config.S3StoragePath(), svc.GerritRepoName)
+	if err := os.RemoveAll(base); err != nil {
+		log.Errorf("Failed to remove dir, err:%s", err)
```

Comment:
If an exception occurs during deletion and does not return, will it affect subsequent operations?